vendor/bat-native-ledger/src/bat/ledger/internal/properties/surveyor_info_properties.h

```diff
@@ -0,0 +1,30 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
```

Comment:
Where is this file used?

Reply:
nvm found it. I think we don't need it as that struct has only property so could be directly used

Reply:
Fixed